docs/src/main/sphinx/release/release-420.md

```diff
@@ -0,0 +1,85 @@
+# Release 420 (21 June 2023)
```

Comment:
Jun

Reply:
It's actually standard to spell it out as June!

Reply:
Nope .. look at all the old ones .. three letter month names everywhere - https://trino.io/docs/current/release.html